### PR TITLE
movieTile and noResults correction 

### PR DESCRIPTION
**Pull request by KrystianGreblowski**
I have made small corrections to MovieTile and ResultsPage.

### CODE REVIEW

File: src/common/Tiles/MovieTilesContainer/MovieTile/index.js [hunk 1]
```diff
@@ -18,26 +17,24 @@ import star from "./star.svg";
 
 export const MovieTile = ({ image, title, year, type, rate, votes }) => {
   return (
-    <TileButton>
```
Comment (Alekzann) [inline src/common/Tiles/MovieTilesContainer/MovieTile/index.js:21]: Got it! I knew that problem with this tiles was so small, but I can't deal with it by myself, SUPER, bravo!

**Review by Alekzann — APPROVED** [review]
Good job! But look carefully, when we reduce the width of the screen, noImagePerson looks smaller than the rest images, can you correct this?
NoMoviePerson has a similar problem, but it all the time is smaller than the rest images.
Try to provide for example "asd" in the search field in both types of searchings and you will see what I mean.
As in your second pull request I approving it, but please look carefully on that things which I had told you.

File: src/common/Tiles/MovieTilesContainer/MovieTile/index.js (hunk 1 above)
Comment (KrystianGreblowski) [reply src/common/Tiles/MovieTilesContainer/MovieTile/index.js:21]: You're right. I haven't seen a problem with noImageMovie and noImagePerson for mobile displays. I'll try to solve it. Thanks

**Comment by KrystianGreblowski** [comment]
You're right. I haven't seen a problem with noImageMovie and noImagePerson for mobile displays. I'll try to solve it. Thanks

**Review by Alekzann — COMMENTED** [review]
I didn't realize that this problem could have such a simple solution, I tried to deal with it in more complicated ways and achieved nothing.... Great job.💪💪💪

**Pull request merged**